Neos.Flow/Scripts/Migrations/Manager.php

```diff
@@ -125,41 +120,22 @@ public function getStatus($packageKey = null, $versionNumber = null)
      * @param boolean $force if TRUE migrations will be applied even if the corresponding package is not a git working copy or contains local changes
      * @return void
      */
-    public function migrate($packageKey = null, $versionNumber = null, $force = false)
+    public function migrate($packageKey, $versionNumber = null, $force = false)
     {
+        if (!$packageKey) {
```

Comment:
I'd say no, it cannot happen; but I wanted to add this additional safeguard. fine for you?

I'll adjust the param annotation.